guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -1,17 +1,50 @@
 [id="glossary-of-terms-used-in-{project-context}_{context}"]
 = Glossary of terms used in {Project}
 
+{Project} is a complete data centre management solution for physical hosts, virtual machines, and cloud instances.
+Key features include automated host provisioning, configuration management, and content management including patch and errata management.
+You can automate tasks and quickly provision hosts, all through a single unified interface.
+
+This alphabetically ordered glossary provides an overview of {Project} related technical terms.
+
 [[Activation_key]]
-Activation key:: A token for host registration and subscription attachment.
-Activation keys define subscriptions, products, content views, and other parameters to be associated with a newly created host.
+Activation key::
+Activation keys are used by `subscription-manager` to register hosts to {Project}.
+They define subscriptions, products, content views, and other parameters to be associated with a newly created host.
++
+Activation keys consist of two parts: a lifecycle environment and a content view.
+They are associated to exactly one lifecycle environment and exactly one content view, though this may be a composite content view.
+You can use them on multiple machines and they behave like configuration information rather than traditional software license keys.
```

Comment:
```suggestion
You can use them on multiple machines and they behave like configuration information rather than traditional software license keys.
You can also use multiple activation keys with a single host.
```